### PR TITLE
chore: update deps, alias mocha as script

### PR DESCRIPTION
**Pull request by marcoscaceres**
This updates everything except jsdom, as latest breaks the build. We can investigate that one separately.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
folks who signed up a as maintainers, @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll, @sangdth ping.

File: package.json [hunk 1]
```diff
@@ -22,24 +22,25 @@
   },
   "main": "./lib/index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha test/",
-    "test-browser": "./node_modules/.bin/mocha test/browser/"
+    "mocha": "mocha",
```
Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [inline package.json]: what is this script for?

Comment (marcoscaceres) [reply package.json]: Otherwise, the script points to "./node_modules/.bin/mocha", which is bad. It let's npm or yarn work out which mocha binary to use.

**Comment by marcoscaceres** [comment]
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll do you have enough rights to do a release?

**Pull request merged**